Source/ACE/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -157,7 +160,7 @@ private void WriteEventBody()
                 }
             }
 
-            var propertiesDid = aceObj.DataIdProperties.Where(x => x.PropertyId < 9000).ToList();
+            var propertiesDid = aceObj.DataIdProperties.Where(x => x.PropertyId < 9000 && x.PropertyValue != null).ToList();
```

Comment:
The issue is that they can be null at runtime.  Null values will ultimately get deleted in the database, but that doesn't stop the game-owned objects from having them.